straxen/plugins/raw_records_coin_nv/nveto_recorder.py

```diff
@@ -79,16 +79,23 @@ class nVETORecorder(strax.Plugin):
         help="Crash if any of the pulses in raw_records overlap with others in the same channel",
     )
 
-    keep_n_chunks_for_monitoring = straxen.URLConfig(
-        default=5,
+    keep_n_seconds_for_monitoring = straxen.URLConfig(
+        default=30,
         track=False,
         infer_type=False,
         help=(
-            "How many chunks at a begining of a run should be "
-            "kept to monitor the detector performance."
+            "Number of seconds which should be stored without applying the software trigger."
+            "Use -1 if all records should be kept without applying the software trigger."
         ),
     )
 
+    run_start = straxen.URLConfig(
+        default="runstart://plugin.run_id?",
+        track=False,
+        infer_type=False,
+        help=("Returns run start in utc unix time in ns."),
```

Comment:
Being picky: `help=("Returns run start in utc unix time in ns."),` -> `help="Returns run start in utc unix time in ns.",`

Reply:
Done